_includes/api-member.html

```diff
@@ -6,6 +6,14 @@ <h2><a class="anchorable" href="#extends">Extends</a></h2>
   {% endfor %}
 {% endif %}
 
+{% if include.data.summary.size > 0 %}
+  {% for summary in include.data.summary %}
+    {% unless include.data.desc contains summary %}
```

Comment:
Without this, for APIs that don't define a `summary` it will duplicate the `desc` (the info in the header) into the page, though it truncates it at the first fullstop. I attempted `!=` too, but that failed, I assume maybe because of the truncation.

Reply:
truncation should only occur on the "index" pages (the ones with the "tiles")